src/panels/config/entities/entity-registry-basic-editor.ts

```diff
@@ -105,6 +141,12 @@ export class HaEntityRegistryBasicEditor extends LitElement {
         .invalid=${invalidDomainUpdate}
         .disabled=${this._submitting}
       ></paper-input>
+      <ha-area-picker
```

Comment:
I think that we should only show it if there is no device linked to this entity.

Reply:
Or maybe we should have it control the device area if it has a linked device.

Reply:
Setting the area of an entity that belongs to a device should be the exception.

Reply:
But then how would you handle the exception?

Reply:
Inside an advanced accordion, with a checkbox that is checked by default "Use Device Area", uncheck it and then the dropdown enables, you can pick it.

Reply:
I don't think the checkbox works, what would you expect to happen when you uncheck the `Use Device Area`, but don't select an area?

Reply:
Maybe like this:
![area-device-entity](https://user-images.githubusercontent.com/5662298/100521424-e668f300-31a3-11eb-8cb5-0bc21bb207b4.gif)